src/ui/components/SearchForm/template.hbs

```diff
@@ -1,6 +1,14 @@
 <div>
-  <form onsubmit={{action submitSearch}} class="search-form">
-    <input value={{search}} oninput={{action updateSearch}} placeholder="Enter location" disabled={{@isDisabled}} class="search-term" data-test-search-input>
+  <form action="/search" onsubmit={{action submitSearch}} class="search-form">
```

Comment:
This should be `method="GET"` - we should never send any `POST` requests to the SSR server.